makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,9 @@
+ALIAS = linux phynode
+
+# this basically implements a look-up table, is there a nicer way in make?
```

Comment:
Maybe [this SO thread](https://stackoverflow.com/a/7285025) could be the start of a solution.

What about:
```
# pba-d-01-kw2x Makefile:
BOARD_ALIAS += phynode

# native Makefile:
BOARD_ALIAS += linux raspberrypi ubuntu

# in alias.inc.mk:
alias.phynode := pba-d-01-kw2x
alias.linux := native

ifneq (,$(filter $(BOARD), $(BOARD_ALIAS)))
  BOARD := ${alias.${BOARD}}
endif
```

Not sure if it works.

Reply:
This won't work per se, as the contents of the board makefiles are not yet known when parsing the alias.inc.mk file. In fact, the board can only be started to be parsed after the alias has been resolved...

But think I like the idea using variable names, will provide another draft of this